services/app-web/src/pages/MccrsId/MccrsId.tsx

```diff
@@ -176,7 +176,10 @@ export const MccrsId = (): React.ReactElement => {
                                     link: `/submissions/${id}`,
                                     text: pkgName || '',
                                 },
-                                { text: 'MC-CRS record number' },
+                                {
+                                    text: 'MC-CRS record number',
+                                    link: `/submissions/${id}/mccrs-record-number`,
```

Comment:
The ticket called out that  "I need the breadcrumb title to clearly call out the page I am on,  for example “MC-CRS record number - Current Page”, so I easily understand where I am on the page.", so in order for this to be tab-able and read out by JAWS I turned it into a link. If there's a better solution please let me know

Reply:
Cool. Let's go ahead and make `link` required then on the Breadcrumbs props itself.  Sounds like there is an a11y impact when we don't ensure every item as a link. That way this doesn't reoccur